packages/grid/src/vaadin-grid-mixin.js

```diff
@@ -855,6 +856,16 @@ export const GridMixin = (superClass) =>
       this._getItem(index, row);
     }
 
+    /** @private */
+    _updateFocusedItem(row) {
```

Comment:
```suggestion
    __updateFocusedItem(row) {
```
We wanted to start using double underscores for private methods.

Reply:
https://polymer-library.polymer-project.org/3.0/docs/devguide/properties#private-and-protected-properties